addons/reorder-custom-inputs/addon.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Rearrangeable custom block inputs",
+  "description": "Adds buttons for rearranging custom blocks inputs",
+  "credits": [
+    {
+      "name": "Chrome_Cat",
+      "link": "https://scratch.mit.edu/users/Chrome_Cat/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"],
+      "runAtComplete": true
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "arrows.css",
+      "matches": ["projects"]
+    }
+  ],
+  "versionAdded": "1.36.0",
+  "tags": ["editor"],
+  "enabledByDefault": true,
```

```suggestion
  "enabledByDefault": false,
```

Sorry, judging by existing addons this wouldn't get enabled by default